source/server/worker_impl.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <memory>
+#include <functional>
```

Comment:
oh! i usually just append, as reordering loses the context of a single addition.